build/steps/build-osx.yml

```diff
@@ -2,12 +2,24 @@ steps:
   - checkout: self
     clean: true
 
+  - task: xamops.azdevex.provisionator-task.provisionator@1
+    displayName: 'Provision Xcode'
+    condition: and(ne(variables['REQUIRED_XCODE'], ''), eq(variables['buildForVS2017'], 'false'))
```

Comment:
What would this variable specify?

Reply:
The enumeration from provisionator so like `Xcode_11_4_1`

https://github.com/xamarin/Xamarin.Forms/pull/10392/files#diff-5c1673f0212d0e8bde062ec627acc073R5